pkg/cmd/server/api/v1/types_test.go

```diff
@@ -118,6 +118,8 @@ imageConfig:
   format: ""
   latest: false
 kind: MasterConfig
+kubeAdmissionControlPluginsOverride:
```

Comment:
> if anywhere, this should be under kubernetesMasterConfig... can you already do this with --admission-control under apiServerArguments?

We want to be able to support this in a stable way.  In addition, when we start a combined API server, our admission controllers will sit in front of both kube and origin.  The name here should probably be `admissionControllerOverride`, but it needs to stay at this level.

Since we need this to be stable, we shouldn't rely upon the unstructured parameters.